### ISSUE TITLE
Define user requirements for timeslice UI upgrade

### ISSUE THREAD

**Issue by timcnicholls** [stfc-aeg/timeslice-camera#1]

**Comment by VKTB** [comment]
Following the meeting with the PE team where we reviewed the requirements for the User Interface (UI) upgrade for the Time-Slice Camera Controller (TSCC), I have now defined and documented all the user requirements:

1.	The UI should work on a Raspberry Pi Touchscreen Setup. 
2.	It should be very user friendly with appropriate colours and as little elements and content as possible.
3.	The elements and the content should be large enough for anyone to see them.
4.	The UI should have a feature that tells the user whether the camera controller is ready to be used or not.
5.	It must have a button which can be tapped by the user to trigger the ‘Capture’ action. 
6.	A visual countdown should be displayed and an audible countdown should begin once the ‘Capture’ action is triggered, giving users enough time to get ready and allowing them to know when exactly the cameras will start capturing.
7.	A loading animated element accompanied by a message should be displayed after the countdown to provide feedback to the user about what is happening in the background. 
8.	Once the video has been created, it should be displayed on a different screen to the Raspberry Pi for the user to see it.
9.	While the video is played on another screen, the UI should display two buttons which the user can tap on. One of the buttons should allow the user to save the video that is displayed and the other should allow the user to capture a new video.
10.	If the user taps the ‘Save video’ button, the video should be saved on the server and an access code as well as a QR code should be generated and displayed on the UI.
11.	The access code should be an alphanumeric randomly generated code that is unique and difficult to guess.
12.	The access code should be linked to the user’s video so that when they write it on the card provided to them, it should form a unique URL which the user can use to download their video on their device at home.
13.	The QR code should automatically direct the user to their video when they scan it using their device.
14.	The video should be automatically deleted from the server after some time to free up disk space.
15.	The UI interface should have a button below the access and the QR code to allow the user to stop displaying the codes when they tap on it.
16.	A message should be displayed straight after the user taps the button located under the codes which tells the user that the video is saved and is ready to be downloaded.
17.	The UI should display the initial screen after the message, allowing the next user to use the TSCC to a capture a video.
